vmslave-config/ansible/deb_build_vmslave_config.yml

```diff
@@ -1,10 +1,10 @@
 ---
 - hosts: deb
   roles:
-    - replace-apt-source
```

Comment:
All prod vmslave is using external apt-ource, and "retry" has been added into "apt-get steps".
So except the "unstable external apt-get", a bigger worry is internal apt source provides different version compare to the external one.
Different version may introduce more problems.
And we are planning to move PR GATE to external network,
Consider all conditions I removed replace-apt-source.